openshift/topics/get_started.adoc

```diff
@@ -1,7 +1,26 @@
 == Get Started
 
 === Using the {project_openshift_product_name} Image Streams and Application Templates
-Red Hat JBoss Middleware for OpenShift images are pulled on demand from the Red Hat Registry: link:https://access.redhat.com/containers/[registry.access.redhat.com]. To update to the latest {project_openshift_product_name} images, run the following commands:
+
+[IMPORTANT]
+====
+Red Hat JBoss Middleware for OpenShift images are pulled on demand from the secured Red Hat Registry: link:https://access.redhat.com/containers/[registry.redhat.io], which requires authentication. To retrieve content, you will need to log into the registry using the Red Hat account.
+
+To consume container images from *_registry.redhat.io_* in shared environments such as OpenShift, it is recommended for an administrator to use a Registry Service Account, also referred to as authentication tokens, in place of an individual's Red Hat Customer Portal credentials.
+
```

Comment:
...in place of an individual person's Red Hat...

Reply:
something like `s/ in place of an individual's Red Hat Customer Portal credentials./, instead of the Red Hat Customer Portal credentials of the individual/`?

Or `, instead of using the Red Hat Customer Portal credentials of the individual`?

Which one do you like better? If none of them, can you suggest a replacement?

Reply:
Your original is better than either of the new options.

I still prefer `in place of an individual person's Red Hat Customer Portal credentials` as it makes it instantly clear that the Registry Service Account is connected to the company.

Reply:
Aha, missed the `person` add-on (read too quickly).

Reply:
Fixed and squashed. Thx!